docs/_docs/plugins.md

```diff
@@ -58,6 +58,21 @@ You have 3 options for installing plugins:
 
 <div class="note info">
   <h5>
+    The <code>:jekyll_plugins</code> group in a <code>Gemfile</code>
```

Comment:
I think I'd prefer if this is not a note and just an added section to the documentation.